gotom-api-doc.yaml

```diff
@@ -192,7 +192,32 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/InternalServerErrorResponse'
-
+                
+  /app-api/campaign/{campaignExternalId}/flight-lineitemid/{flightExternalId}:
+    get:
+      tags: [CampaignFlight]
+      summary: Retrieve the flight lineItemIDs and strategyIDs if flight is connected with adServer(s)
```

Comment:
Retrieve the Flight LineItem Ids and Strategy Ids if it is connected with an AdServer